sky/skylet/events.py

```diff
@@ -77,62 +77,74 @@ def _run(self):
 
 
 class AutostopEvent(SkyletEvent):
-    """Skylet event for autostop."""
+    """Skylet event for autostop.
+
+    Semantics:
+    - Idleness timer gets reset whenever
+      - A first autostop setting is set. By "first", we mean either there's
+        never any autostop setting set, or the last autostop setting is a
+        cancel (idle minutes < 0).
+      - Or, whenever this event wakes up and job_lib.is_cluster_idle() returns
+        False.
+      - TODO(zongheng): what about if the cluster has restarted? The
+        autostop_config.boot_time != psutil.boot_time() check below.
```

Comment:
The `autostop` set before the cluster is restarted will be deactivated.

Reply:
Thanks, fixed the comment.